tests/low-disk/create_and_search_items.py

```diff
@@ -67,22 +95,78 @@ def search_point(qdrant_host, collection_name):
     return resp.json()
 
 
-def initialize_qdrant(qdrant_host, collection_name, points_amount):
-    create_collection(qdrant_host, collection_name)
+def insert_points_and_search(qdrant_host, collection_name, points_amount):
+    collection_json = {
+        "vectors": {
+            "size": VECTOR_SIZE,
+            "distance": "Cosine"
+        }
+    }
+    create_collection(qdrant_host, collection_name, collection_json)
     for points_batch in generate_points(points_amount):
         insert_points(qdrant_host, collection_name, points_batch)
         search_point(qdrant_host, collection_name)
```

Comment:
Would it make more sense to insert all points first, and search after, using two separate loops? Or is this order required for this test?

I know this has not been changed in this PR, but I only noticed this now.

Reply:
I don't think we even need a loop for the search.